src/Psalm/Type/Atomic/TLiteralString.php

```diff
@@ -24,15 +24,18 @@ public function getKey(bool $include_extra = true): string
         return 'string(' . $this->value . ')';
     }
 
-    public function getId(bool $nested = false): string
+    public function getId(bool $exact = true, bool $nested = false): string
     {
+        if (!$exact) {
+            return 'string';
+        }
         // quote control characters, backslashes and double quote
         $no_newline_value = addcslashes($this->value, "\0..\37\\\"");
         if (mb_strlen($this->value) > 80) {
-            return '"' . mb_substr($no_newline_value, 0, 80) . '...' . '"';
+            return "'" . mb_substr($no_newline_value, 0, 80) . '...' . "'";
         }
 
-        return '"' . $no_newline_value . '"';
+        return "'" . $no_newline_value . "'";
```

Comment:
@orklah Was this an intentional change? It caused one of my tests to fail when rebasing because `$foo = "bar"` now has type `'bar'` instead of `"bar"`.

Reply:
This was indeed intended. I mentionned it in the PR
> getId() on TLiteralString was using double quotes to display the types. This is not consistent with the rest of the types (in docblock for example) so I changed that

I didn't thought about plugins, I'll add this in UPGRADE file

Reply:
https://github.com/vimeo/psalm/commit/efe9c2bc6d55f532b69c63324e35dd98f8139dfd